packages/flutter/test/gestures/monodrag_test.dart

```diff
@@ -10,6 +10,31 @@ import 'gesture_tester.dart';
 void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
 
+  test('acceptGesture tolerates a null lastPendingEventTimestamp', () {
+    // Regression test for https://github.com/flutter/flutter/issues/112403
+    // and b/249091367
+    final DragGestureRecognizer recognizer = VerticalDragGestureRecognizer();
+    const PointerDownEvent event = PointerDownEvent(timeStamp: Duration(days: 10));
+
+    expect(recognizer.debugLastPendingEventTimestamp, null);
```

Comment:
In theory, this test would work without exposing (and checking) debugLastPendingEventTimestamp. If the bug were to be re-introduced, it would still crash on line 34, no?

I think I would prefer that over exposing the debug field.

Reply:
It would crash, but it wouldn't actually be asserting anything besides that methods can be called.

My main concern over not exposing the field is that someone would be able to easily refactor the internals of the class and the test would still pass but they'd re-introduce the failure case. This actually asserts that `acceptGesture` can tolerate that field being null.